libs/content/workspace/src/tab/markdown_editor/input/cursor.rs

```diff
@@ -150,7 +150,6 @@ pub struct PointerState {
     /// Type, position, modifiers, and drag status of current click, recorded on press and processed on release
     pub click_type: Option<ClickType>,
     pub click_pos: Option<Pos2>,
-    pub click_offset: Option<DocCharOffset>,
```

Comment:
The pointer state shouldn't store the click offset because it changes based on text layout. The click offset can be reproduced using the click position and text layout together.